### ISSUE TITLE
Required PHP extension 'Imagick' was not loaded

### ISSUE THREAD

**Issue by kenbradleywhite** [markshust/docker-magento#530]
**Description**

Getting the "Required PHP extension 'Imagick' was not loaded" error after adding a new theme.

The problem goes away when deleting the theme (although obviously I'm setting up a new theme)

**Steps To Reproduce**
1. Create a new empty theme in Magento's app/design folder - just the usual boilerplate files
2. Run `setup:upgrade`

**Expected Result**

It setup:upgrades

**Actual Result**

```
...
Module 'Magento_AdobeStockImageApi':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Directory':
Module 'Magento_Amqp':
Module 'Magento_AmqpStore':
Module 'Magento_Config':
Module 'Magento_Theme':
Running data recurring...Required PHP extension 'Imagick' was not loaded.
```

**Comment by kenbradleywhite** [comment]
I've worked around this by manually adding the new theme to the `theme` table

**Comment by markshust** [comment]
@kenbradleywhite which PHP image/tag are you using?

**Comment by markshust** [comment]
I'm going to close this as I'm unable to replicate, please re-comment if you have further instructions.

**Comment by markshust** [comment]
Released in version `40.1.0`
